### PR TITLE
EIPS eligible for Inclusion

### PR DESCRIPTION
**Pull request by MadeofTin**
Meta EIP for tracking Eligibility for Inclusion as defined in the EIP-centric Forking Model.

### CODE REVIEW

File: EIPS/EIP-xxx.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+eip: <to be assigned>
+title: EIPS eligible for Inclusion
```
Comment (axic) [inline EIPS/EIP-xxx.md]: I suggest "EIPs Eligible for Inclusion"

File: EIPS/EIP-xxx.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+author: James Hancock (MadeofTin)
+discussions-to: <URL>
+status: Draft
```
Comment (axic) [inline EIPS/EIP-xxx.md]: Didn't we say this is in Active status (e.g. a registry)?

File: EIPS/EIP-xxx.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+---
+eip: <to be assigned>
+title: EIPS eligible for Inclusion
+author: James Hancock (MadeofTin)
```
Comment (axic) [inline EIPS/EIP-xxx.md]: Invalid format (needs the @)

File: EIPS/EIP-xxx.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+discussions-to: <URL>
+status: Draft
+type: Meta
+
```
Comment (axic) [inline EIPS/EIP-xxx.md]: Stray empty line

File: EIPS/EIP-xxx.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+eip: <to be assigned>
```
Comment (axic) [inline EIPS/EIP-xxx.md]: 2378

File: EIPS/EIP-xxx.md (hunk 2 above)
Comment (MadeofTin) [reply EIPS/EIP-xxx.md]: Should it go directly to Active? Or start as Draft and then be moved to Active?

**Comment by timbeiko** [comment]
Could we merge this? Have been linking it to multiple people as the "official" resource of EIPs that are being considered. cc: @axic @MadeofTin

File: EIPS/eip-2378.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+| EIP-1380 | Reduced gas cost for call to self                     | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+| EIP-1559 | Fee market change for ETH 1.0 chain                   | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+| EIP-1702 | Generalized Account Versioning Scheme                 | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+| EIP-1962 | EC arithmetic and pairings with runtime definitions   | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+| EIP-1985 | Sane limits for certain EVM parameters                | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+| EIP-2046 | Reduced gas cost for static calls made to precompiles | ELIGIBLE | 2019-11-01 | [🔗](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2074.md) |
+
+EIP-1057
```
Comment (timbeiko) [inline EIPS/eip-2378.md]: Duplicate of line 46

Comment (MadeofTin) [reply EIPS/eip-2378.md]: removed

**Comment by axic** [comment]
> Could we merge this? Have been linking it to multiple people as the "official" resource of EIPs that are being considered.

We have agreed in December with @MadeofTin that this EIP should not propose a change in process (as the process just should be part of EIP-1 or EIP-233, because having multiple draft process EIPs is extremely confusing) and be only a listing. Was this change done?

**Comment by MadeofTin** [comment]
Not yet. Adding it to EIP-1 is part of the plan of the EIPIP work, but Tim mentioned that not having a place to point people to see the registry has made working with it difficult. In my view the process changed when the ACD agreed to move to the EIP-Centric model. The description in 2378 was meant to be informational to those who don't know anything about the model. When I made it just as a registry it was unclear. The description in 2378 isn't sufficient to describe the actual process change which should take place in EIP-1. 

Pulling the description out of here, before having the process change described in EIP in my view would add more confusion. As EIP-1 progresses we can remove it from here. I am still working on what is the best way to add it to EIP-1.

**Comment by MadeofTin** [comment]
IMO: we should keep2378 as DRAFT until EIP-1 is updated and then be moved to ACTIVE.

**Pull request merged**